docs/source_files/pixi_projects/pixi_build_cpp/pixi.toml

```diff
@@ -21,9 +21,6 @@ channels = [
 ]
 
 [package.host-dependencies]
-cmake = ">=3.20, <3.27"      # (8)!
-nanobind = ">=2.4.0, <2.5.0" # (6)!
-python_abi = ">=3.12, <3.13" # (7)!
-
-[package.run-dependencies]
-python = ">=3.12, <3.13" # (9)!
```

Comment:
You still need run dependencies to be able to install only this package and be able to run it in this example.

Reply:
My understanding is that python adds a run export but I could be wrong!

Reply:
time to render run exports on prefix :)

Reply:
thats a fun little project!

Reply:
Without the `run-dependencies` it doesn't work for me, so I'm not sure if something else is broken, but it was a finding while trying the tutorials